Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -440,7 +440,19 @@
             <trans-unit id="form.label_provider_reference">
                 <source>form.label_provider_reference</source>
                 <target>Provider Referenz</target>
-            </trans-unit>            
+            </trans-unit>
+            <trans-unit id="sonata.media.block.media">
+                <source>sonata.media.block.media</source>
+                <target>Medium</target>
```

Comment:
As @rande said. The German singular is for *Medien* is *Medium* 
http://www.duden.de/rechtschreibung/Medium_Vermittler_Traeger